### PR TITLE
Python

### PR DESCRIPTION
**Pull request by jdonszelmann**
Added a small python server which both statically serves the dist folder and simultaniously serves 2 ipv8 peers

### CODE REVIEW

**Comment by jdonszelmann** [comment]
oh wait

**Comment by jdonszelmann** [comment]
should we merge into the other branch and merge them together into master?

**Comment by dsluijk** [comment]
I think it's best to merge into `feat/boilerplating` first, then merge into `master`, as this model does not really make sense anymore. After merging to `master` we can start using feature branches

File: Pipfile.lock [hunk 1]
```diff
@@ -0,0 +1,278 @@
+{
```
Comment (dsluijk) [inline Pipfile.lock]: You could probably delete and ignore this file, as it generates a lot of noise.

**Review by dsluijk — CHANGES_REQUESTED** [review]

**Review by dsluijk — APPROVED** [review]

**Pull request merged**